test/libs/blocks/faas/faas.test.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
Should we have any additional test cases?

Reply:
Since all the work is done by faas/utils.js I think this is good enough for now.